COMMUNITY-MANAGER.md

```diff
@@ -0,0 +1,45 @@
+# Dapr Community Managers
+
+## Charter
+
+Community Managers are responsible for managing many aspects of the Dapr community. The role is to engage and promote the Dapr community and have someone that the community can reach out to for guidance or to raise a concern. This document outlines a list of duties that a community manager can perform.
+
+### Duties
+
+A Community Manager can perform the following duties:
+
+* Managed the Discord community including posting annoucements, organizing and monitoring the Discord channels, establishing community rules, and aiding on the overall health of the Discord community. They are included in the Discord Community Manager role.
```

Comment:
I feel we're clubbing too many things in this discord one.  
Health feels like a great one for community manager to monitor and manage in conjunction with maintainers. 
Harvesting themes/ideas and recruiting community members to discuss in higher forums (community call, blogs, or GH issues) feels like a potential activity for community manager.   
Posting announcements feels like a marketing or product type activity, and I'm not sure that's expected duty. 
Responding to or moderating discord questions often requires skillsets closer to maintainers and approvers, and again I'm not sure that's expected duty unless the community manager builds those skills and passion on a given discord channel.

Reply:
Not expected to respond to questions on the forums. However there have been times when people asked for a new channel or an new channel (like Workflow) had to be created and currently this burden is on the STC members. Discord is a primary engagement to the community here.

Reply:
s/annoucements/announcements/

Reply:
Removed the announcements and kept this to monitoring the Discord community health